templates/aurora_mysql-main.template.yaml

```diff
@@ -240,7 +240,7 @@ Parameters:
     Type: String
   EnableTCPForwarding:
     Type: String
-    Description: With TCP forwarding on the bastion host, you can set up traffic forwarding to resources in private subnets. To enable TCP forwarding, choose "true." This setting enables only the capability; it doesn't set up any traffic forwarding. 
+    Description: With TCP forwarding on the bastion host, you can set up traffic forwarding to resources in private subnets. To enable TCP forwarding, choose "true". This setting enables only the capability; it doesn't set up any traffic forwarding. 
```

Comment:
@catcharbind I understand the logic of your suggestion. We follow the AWS Style Guide, though. It says to follow the American English standard of putting the period inside the quotation mark. The only time we would avoid doing so would be if this were a text string that people enter. In that case, it would be confusing to include the period since people might type it.  For a dropdown option, there's no danger of confusion. (See "quotation marks" in the [AWS Style Guide](https://alpha-docs-aws.amazon.com/awsstyleguide/latest/styleguide/punctuation-topic.html).)

Reply:
Makes sense. Taken care now.